src/Onboarding/Setup/Handlers/StripeConnectHandler.php

```diff
@@ -0,0 +1,103 @@
+<?php
+
+/**
+ * Onboarding class
+ *
+ * @package Give
+ */
+
+namespace Give\Onboarding\Setup\Handlers;
+
+defined( 'ABSPATH' ) || exit;
+
+/**
+ * Fork of `give_stripe_connect_save_options()`
+ */
+class StripeConnectHandler implements RequestHandler {
+
+	public function maybeHandle() {
+
+		// Is user have permission to edit give setting.
```

Comment:
@kjohnson This function runs specifically to save Stripe details when admin returns from the connect server. Can you use Give_Admin_Setting::is_setting_page to validate specific tab and section.

Reply:
This one is a fork of the same logic and isn't on a setting page.